app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -116,10 +116,11 @@ fun PocketStory(
         title = {
             Text(
                 text = story.title,
+                modifier = Modifier.testTag("homescreen.pocket.story.title"),
                 color = FirefoxTheme.colors.textPrimary,
-                fontSize = 14.sp,
                 overflow = TextOverflow.Ellipsis,
                 maxLines = 2,
+                style = FirefoxTheme.typography.caption,
```

Comment:
It seems odd to me that we would use Caption for the story header.

Reply:
Thank you.
Discussed with UX who updated the design to use `body2` for titles and `caption` elsewhere.